pkg/minikube/driver/driver.go

```diff
@@ -85,6 +85,14 @@ func Supported(name string) bool {
 	return false
 }
 
+// MachineType returns appropriate machine name for the driver
+func MachineType(name string) string {
```

Comment:
@nkprince007 can we have a test for this func? :)

Reply:
Sure :)